docker/files/etc/prosody/prosody.cfg.lua

```diff
@@ -39,7 +39,9 @@ use_libevent = true
 
 
 cross_domain_bosh = true
+cross_domain_websocket = true
 consider_bosh_secure = true
+consider_websocket_secure = true
```

Comment:
I was wondering, your commit message is saying "instead of bosh system", but in the configuration file, bosh is still enabled.

Reply:
Yes bosh is still enabled but used only if you want. You can use bosh or websocket and this configuration is only for development purpose so we need to keep it. This is not the case in production.

Reply:
> We can connect to a xmpp server using a websocket url. Using bosh url is still allowed.

My first commit message.